src/components/flightPlan/flightPlan.js

```diff
@@ -98,28 +101,32 @@ export default function FlightPlan({ flightPlanData, onSavePlan }) {
         <FlightPlanInformation>
           <h2>Flight Plan Information</h2>
           <p>Flight Number: {flightPlanData.id}</p>
-
           <FligthPlanDetails>
             <DepartureLogo />
-            <p>
-              Departure Airport: {flightPlanData.departureAirport} from Runway{" "}
-              {flightPlanData.departureRunway}
-            </p>
+            <p>Departure Airport: {flightPlanData.departureAirport}</p>
+            <p>from Runway:{flightPlanData.departureRunway}</p>
           </FligthPlanDetails>
           <FligthPlanDetails>
             <ArrivalLogo />
-            <p>
-              Arrival Airport: {flightPlanData.arrivalAirport} at Runway:{" "}
-              {flightPlanData.arrivalRunway}
-            </p>
+            <p>Arrival Airport: {flightPlanData.arrivalAirport}</p>
+            <p>at Runway: {flightPlanData.arrivalRunway}</p>
           </FligthPlanDetails>
-
           <FligthPlanDetails>
             <DateandTimeLogo />
-            <p>
-              Flight Date: {flightPlanData.flightDate} Takeoff Booked Time:{" "}
-              {flightPlanData.flightTime}
-            </p>
+            <input
+              id="flightDateFP"
+              type="date"
+              defaultValue={flightPlanData.flightDate}
+              min={new Date().toISOString().split("T")[0]}
+              onChange={(event) => setFlightDate(event.target.value)}
+            />
+            <input
+              id="flightTimefP"
```

Comment:
Should this be`flightTimeFP`? Just asking cuz its `flightDateFP` on the date input. Just to be uniform :)